endpoints/openrtb2/video_auction_test.go

```diff
@@ -363,6 +363,119 @@ func TestVideoEndpointValidationsPodErrors(t *testing.T) {
 	assert.Equal(t, "request missing or incorrect required field: PodConfig.Pods.ConfigId, Pod index: 3", podErrors[1].ErrMsgs[2], "Pod error ind 1 should have missing config id")
 }
 
+func TestVideoBuildVideoResponseMissedCacheForOneBid(t *testing.T) {
+	openRtbBidResp := openrtb.BidResponse{}
+	podErrors := make([]PodError, 0)
+
+	seatBids := make([]openrtb.SeatBid, 0)
+	seatBid := openrtb.SeatBid{}
+
+	bids := make([]openrtb.Bid, 0)
+	bid1 := openrtb.Bid{}
+	bid2 := openrtb.Bid{}
+	bid3 := openrtb.Bid{}
+
+	extBid1 := []byte(`{"prebid":{"targeting":{"hb_bidder":"appnexus","hb_pb":"17.00","hb_pb_cat_dur":"17.00_123_30s","hb_size":"1x1","hb_uuid":"837ea3b7-5598-4958-8c45-8e9ef2bf7cc1"}}}`)
+	extBid2 := []byte(`{"prebid":{"targeting":{"hb_bidder":"appnexus","hb_pb":"17.00","hb_pb_cat_dur":"17.00_456_30s","hb_size":"1x1","hb_uuid":"837ea3b7-5598-4958-8c45-8e9ef2bf7cc1"}}}`)
+	extBid3 := []byte(`{"prebid":{"targeting":{"hb_bidder":"appnexus","hb_pb":"17.00","hb_pb_cat_dur":"17.00_406_30s","hb_size":"1x1"}}}`)
+
+	bid1.Ext = extBid1
+	bids = append(bids, bid1)
+
+	bid2.Ext = extBid2
+	bids = append(bids, bid2)
+
+	bid3.Ext = extBid3
+	bids = append(bids, bid3)
+
+	seatBid.Bid = bids
+	seatBids = append(seatBids, seatBid)
+	openRtbBidResp.SeatBid = seatBids
+
+	bidRespVideo, err := buildVideoResponse(&openRtbBidResp, podErrors)
+	assert.Equal(t, nil, err, "Error shopuld be nil")
```

Comment:
`testify/assert` package also has a [`assert.NoError(t, err)`](https://godoc.org/github.com/pgpst/pgpst/internal/github.com/stretchr/testify/assert#NoError) assertion available. Not saying it needs to be changed in this PR to use this but just thought might be interesting to know.

Also, similar for length check below: [`assert.Len(t, bidRespVideo.AdPods, 1, "err msg")`](https://godoc.org/github.com/pgpst/pgpst/internal/github.com/stretchr/testify/assert#Assertions.Len)

Reply:
Good point! Changed!